site/public/js/grade-inquiry.js

```diff
@@ -1,5 +1,25 @@
 /* global buildCourseUrl, WebSocketClient */
-/* exported initGradingInquirySocketClient, onComponentTabClicked, onGradeInquirySubmitClicked, onReady, onReplyTextAreaKeyUp */
+/* exported saveDraft, initGradingInquirySocketClient, onComponentTabClicked, onGradeInquirySubmitClicked, onReady, onReplyTextAreaKeyUp */
+
+function saveDraft() {
+    //saveDraft function displays the saved draft text on grade inquiry box
+    let draftContent = localStorage.getItem('content');
+    if (draftContent === null) {
+        draftContent = {};
+    }
+    else {
+        draftContent = JSON.parse(draftContent);
+    }
+    const elements = document.getElementsByClassName('markdown-textarea fill-available ');
+    for (let i = 0; i < elements.length; i++) {
```

Comment:
You can use [`.forEach()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach) for this instead.